app/src/main/java/io/token/sample/Application.java

```diff
@@ -164,8 +231,17 @@ private static Member initializeMember(TokenClient tokenClient) {
         // A member's profile has a display name and picture.
         // The Token UI shows this (and the alias) to the user when requesting access.
         member.setProfile(Profile.newBuilder()
-                .setDisplayNameFirst("Info Demo")
+                .setDisplayNameFirst("Info")
```

Comment:
Should this be "PFM Demo" or something? basically it should be consistent with the other samples.